samples/matter/common/src/persistent_storage/persistent_storage.h

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "persistent_storage_common.h"
+
+namespace Nrf
+{
+
+class PersistentStorageImpl;
+
+/**
+ * @brief Interface class to manage persistent storage using generic PersistentStorageNode data unit
+ */
+class PersistentStorage {
+public:
+	/**
+	 * @brief Load all data from persistent storage
+	 *
+	 * @return true if data has been loaded
+	 * @return false otherwise
+	 */
```

Comment:
Why does the `Init` method do the same operations as the `Load` method? It looks like there is something hidden under the Init method, or the Init method just calls Load multiple times.